packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -46,17 +43,17 @@ export const PopupMessages: React.FunctionComponent = ( { children } ) => {
 	);
 };
 
-type ToggleHostProps = {
-	intervalType: BillingIntervalType;
-	onChange: ( selectedValue: BillingIntervalType ) => void;
-	maxSavingsPerc: number;
+interface PlansIntervalToggleProps {
+	intervalType: Plans.PlanBillingPeriod;
+	onChange: ( selectedValue: Plans.PlanBillingPeriod ) => void;
+	maxMonhtlyDiscountPercentage?: number;
 	className?: string;
-};
+}
 
-const PlansIntervalToggle: React.FunctionComponent< ToggleHostProps > = ( {
+const PlansIntervalToggle: React.FunctionComponent< PlansIntervalToggleProps > = ( {
 	onChange,
 	intervalType,
-	maxSavingsPerc,
+	maxMonhtlyDiscountPercentage,
```

Comment:
```suggestion
	maxMonthlyDiscountPercentage,
```

Reply:
It took be 2 minutes of staring at this suggestion to understand the typo 🤦  I'll have to refactor the code in a few places — nice catch!

Reply:
I highly recommend [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)! Can save you from bugs too (in JS). 

![image](https://user-images.githubusercontent.com/17054134/105857515-a9c24800-5fea-11eb-94e9-e672ea0c9938.png)

Reply:
🙏

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/146a5b7927b5093cb2803108bba8dee4bf853365